src/Column.vue

```diff
@@ -76,31 +120,47 @@ export default {
   max-width: var(--xsWidthVGR);
 }
 
+.colOffset-xs {
+  margin-left: var(--xsOffset);
+}
+
 @media (min-width: 576px) {
   .colVGR-sm {
     flex-basis: var(--smWidthVGR);
     max-width: var(--smWidthVGR);
   }
+  .colOffset-sm {
```

Comment:
I was interested in leaving all styles with the acronym VGR(vue grid resposive). Would it be possible to make this change?

Reply:
Sure. How about 'colVGR-Offset-sm/xs/.. ?

Reply:
Great!!!

Reply:
Done